docs/how-to/how-to-run-dind-with-kata.md

```diff
@@ -0,0 +1,122 @@
+# How to run Docker in Docker with Kata Containers
+
+This document describes the why and how behind running Docker-in-Docker ("dind") in a kata container.
+
+## Requirements
+
+- A working Kata Containers installation
+
+## Install and configure Kata Containers
+
+Follow the [Kata Containers installation guide](../docs/install/README.md) to Install Kata Containers on your Kubernetes cluster.
+
+## Background
+
+Docker in Docker ("DinD") is the colloquial name for the ability to run `docker` from inside a container (whether provided by `docker` or otherwise) pattern for running `docker` from *inside* an existing `docker` container. You can read more about Docker-in-docker at the following links:
+
+- [The original announcement of DinD](https://www.docker.com/blog/docker-can-now-run-within-docker/)
+- [`docker` image DockerHub page](https://hub.docker.com/_/docker/) (this page lists the `-dind` releases)
+
+**NOTE** While in the past kata containers did not contain the [`overlay` kernel module (aka OverlayFS)][overlayfs], it *now does* with the merging of the following pull requests:
+
+- [PR #1384](https://github.com/kata-containers/kata-containers/pull/1384)
+- [PR #716](https://github.com/kata-containers/packaging/pull/716)).
+
+`overlayfs` is the preferred containerization storage driver for linux systems ([Docker in particular](https://docs.docker.com/storage/storagedriver/select-storage-driver)), and as such, using it is often desired over other alternatives.
```

Comment:
```suggestion
`OverlayFS` is the preferred containerization storage driver for Linux systems ([Docker in particular](https://docs.docker.com/storage/storagedriver/select-storage-driver)), and as such, using it is often desired over other alternatives.
```

overlayfs -> OverlayFS is a suggestion of mine as what we're adding to the dict in the PR that I mentioned is OverlayFS.

linux -> Linux is a suggestion from the spell-checker.

Reply:
Perfect I'll add this too:

- [ ] `overlayfs` -> `OverlayFS` on line 25
- [ ] "linux" -> "Linux" on line 25

Reply:
oh also here I think OverlayFS is more the project/effort name and not a binary in particular so I'll drop the backticks